content/commands/ts.alter/index.md

```diff
@@ -65,7 +65,7 @@ stack_path: docs/data-types/timeseries
 summary: Update the retention, chunk size, duplicate policy, and labels of an existing
   time series
 syntax: 'TS.ALTER key [RETENTION retentionPeriod] [CHUNK_SIZE size] [DUPLICATE_POLICY
-  policy] [LABELS [{label value}...]] '
+  policy] [LABELS {label value}...] '
```

Comment:
I prefer the way it's written in the syntax_fmt block, if that works for you:

```suggestion
  policy] [LABELS\_label value [label value ...]'
```

Reply:
@dwdougherty Your suggested syntax is not known to me. Whatever you find appropriate, feel free to go ahead.

On a relevant note, TS.ALTER should have same syntax as [TS.CREATE](https://github.com/redis/docs/blob/edda9a5364a2a4998c16cdb7de327fe7ac9307f7/content/commands/ts.create/index.md?plain=1#L77) and [TS.ADD](https://github.com/redis/docs/blob/edda9a5364a2a4998c16cdb7de327fe7ac9307f7/content/commands/ts.add/index.md?plain=1#L81). My suggestion here for TS.ALTER is the current syntax in both the other mentioned commands.

Reply:
Thanks, @sazzad16 . I'll go with your first suggestion.